docs/runbooks/NoReadyVirtController.md

```diff
@@ -2,51 +2,52 @@
 
 ## Meaning
 
-From a high-level perspective the virt-controller is responsible for all the cluster wide virtualization functionality.
+This alert triggers when no available `virt-controller` devices have been detected for 5 minutes.
```

Comment:
We have to use "fires" (I don't like the word) as in "this alert fires ..." because that is the actual state in the UI. 
Spell out numbers less than "10": "five minutes"